api-reference/v1.0/resources/groups_overview.md

```diff
@@ -0,0 +1,273 @@
+# Working with groups in Microsoft Graph

```

Comment:
So just a general comment, I think we need follow the format for overview topics that Angela and Laura have been working on. Good example is here:  https://developer.microsoft.com/en-us/graph/docs/api-reference/v1.0/resources/mail_api_overview with the addition of an Auth section similar to what is in the AAD overview topics: https://developer.microsoft.com/en-us/graph/docs/api-reference/v1.0/resources/azure_ad_overview#authorization .